packages/app/src/update/services/device-update.service.ts

```diff
@@ -53,6 +54,19 @@ export class DeviceUpdateService {
       )
     }
     this.beforeUpdateOsVersion = deviceInfoResult.data.osVersion
+    if (
+      isVersionGreaterOrEqual(
+        this.beforeUpdateOsVersion,
+        payload.fileName.split("-")[1]
```

Comment:
suggest to store it in variable 'payload.fileName.split("-")[1]' and then pass to function for clarity :)